compiler/luci/lang/src/Nodes/CircleAbs.test.cpp

```diff
@@ -29,3 +32,63 @@ TEST(CircleAbsTest, constructor)
 
   ASSERT_EQ(nullptr, abs_node.x());
 }
+
+TEST(CircleAbsTest, common_NEG)
+{
+  luci::CircleAbs abs_node;
+
+  abs_node.name("name");
+  ASSERT_EQ("name", abs_node.name());
+
+  auto q = std::make_unique<luci::CircleQuantParam>();
+  abs_node.quantparam(std::move(q));
+  ASSERT_NE(nullptr, abs_node.quantparam());
+
+  ASSERT_EQ(luci::ShapeStatus::UNDEFINED, abs_node.shape_status());
+  abs_node.shape_status(luci::ShapeStatus::NOSHAPE);
+  ASSERT_NE(luci::ShapeStatus::UNDEFINED, abs_node.shape_status());
+}
+
+TEST(CircleAbsTest, input_NEG)
+{
+  luci::CircleAbs abs_node;
+  luci::CircleAbs node;
+
+  abs_node.x(&node);
+  ASSERT_NE(nullptr, abs_node.x());
+
+  abs_node.x(nullptr);
+  ASSERT_EQ(nullptr, abs_node.x());
+}
+
+TEST(CircleAbsTest, arity_NEG)
+{
+  luci::CircleAbs abs_node;
+
+  ASSERT_NO_THROW(abs_node.arg(0));
+  ASSERT_THROW(abs_node.arg(1), std::out_of_range);
```

Comment:
I am not sure `std::out_of_range` should come first or not like `ASSERT_EQ(nullptr, abs_node.x());`.
If this is right, ignore this comment :)

Reply:
From gtest,
```cxx
#define ASSERT_THROW(statement, expected_exception)
```
throw is not like others... strange :)

Reply:
@seanshpark Then my comment was wrong :) Thank you for detailed information!